omnibus/config/software/python3.rb

```diff
@@ -59,18 +59,20 @@
   end
 
 else
-  dependency "vc_redist_14"
-
+  #
+  # note for next version after 3.8.1, remove the `-withcrt` as the filename won't
```

Comment:
We will still need to include the runtime in the python zip for every next version from now on, right? The process of generating this zip is becoming increasingly difficult so please make sure it is documented somewhere including this new step.

Reply:
well, it gets _less_difficult because the old process explicitly removed the runtime (to not conflict with the merge module) and the new one doesn't.  But, yes.. point is taken.

Reply:
Oh, that's good new then :)